### DISCUSSION TITLE
How fast can you get GFPgan? Does it use GPU?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [TencentARC/GFPGAN#478] [Q&A]
Hello, I have been founding GFPgan to be a bit slow, it might possibly explained by the fact I am not using GPU much?
Could someone share a screenshot of their GPU usage when running GFPgan? Can you tell me how fast you can process 10 sec or 1 minute video? (tell me the resolution of the video)
thanks

**Comment by jay7000911882** [comment]
![IMG20240201141247](https://github.com/TencentARC/GFPGAN/assets/123394013/796eb831-d529-4606-bf4d-db424bd33cb2)
3D

**Reply by AIhasArrived** [reply]
?
